collectors/ebpf.plugin/README.md

```diff
@@ -76,7 +76,7 @@ collector uses the following `tracepoints` and `kprobe`:
 
 - `sched/sched_process_fork`: Tracepoint called after a call for `fork (2)`, `vfork (2)` and `clone (2)`.
 - `sched/sched_process_exec`: Tracepoint called after a exec-family syscall.
-- `kprobe/kernel_clone`: This is the main [fork](https://elixir.bootlin.com/linux/v5.10/source/kernel/fork.c#L2415) 
+- `kprobe/kernel_clone`: This is the main [`fork()`](https://elixir.bootlin.com/linux/v5.10/source/kernel/fork.c#L2415)
```

Comment:
@DShreve2 considering we have a link here, do we also need to add back quotes?

Reply:
Let's go no back quotes just to avoid confusion.

Reply:
When I preview it, it looks like back quotes still renders it clearly as a link and element, so I'm okay with both.